bond/bond.py

```diff
@@ -158,6 +160,20 @@ def getDeviceState(self, deviceId):
         r = requests.get(url, headers=headers)
         return r.json()
 
+    def getDeviceCommandIds(self, deviceId):
+        url = f"http://{self.bondIp}/v2/devices{deviceId}/commands"
```

Comment:
Are we missing a / here?

Reply:
No, it does not need to end with /, nut if you prefer I can add. LMK

Reply:
good catch, thanks! fixed.